features/upgrade/monitoring/monitoring-upgrade.feature

```diff
@@ -0,0 +1,82 @@
+Feature: cluster monitoring related upgrade check
+  # @author hongyli@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: upgrade cluster monitoring along with OCP - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :apply client command with:
+      | f          | <%= BushSlicer::HOME %>/features/upgrade/monitoring/testdata/cm-monitoring-retention.yaml |
+      | overwrite  | true |
```

Comment:
@lihongyan1 table alignment

Reply:
In many feature files, if one parameter is very long, we do alignment like this.